src/inbox/mod_inbox.erl

```diff
@@ -267,6 +271,12 @@ remove_user(Acc, User, Server) ->
     mod_inbox_utils:clear_inbox(User, Server),
     Acc.
 
+-spec remove_domain(mongoose_acc:t(), mongooseim:host_type(), jid:lserver()) ->
```

Comment:
it's not `mongoose_acc:t()`, MB we need some special type to highlight that Acc is not used!

Reply:
ok, changed.